src/components/Header.tsx

```diff
@@ -173,24 +173,21 @@ export const Header: React.FC = () => {
                   as={NavLink}
                   className={classnames(styles["nav-link"], {
                     [`fw-bold ${styles.active}`]: useIsActive(
-                      ["/gallery", "/oer"],
+                      ["/catalog"],
                       false
                     ),
                   })}
                 >
                   Catalog
                 </Dropdown.Toggle>
                 <Dropdown.Menu>
-                  <NavDropdownItem href="/gallery/questions">
+                  <NavDropdownItem href="/catalog/questions">
                     Questions
                   </NavDropdownItem>
-                  <NavDropdownItem href="/gallery/assessments">
+                  <NavDropdownItem href="/catalog/assessments">
                     Assessments
                   </NavDropdownItem>
-                  <NavDropdownItem href="/gallery/courses">
-                    Courses
-                  </NavDropdownItem>
-                  <NavDropdownItem href="/oer">OER</NavDropdownItem>
+                  <NavDropdownItem href="/catalog/oer">OER</NavDropdownItem>
```

Comment:
Should we call this "Open Educational Resources (OER)"? I worry that people will ignore the menu item with a possibly-obscure acronym.

Reply:
updated to "Open Educational Resources". Left the OER abbreviation to only appear at the top of the page.